tests/DCPS/WriteDataContainer/WriteDataContainerTest.cpp

```diff
@@ -350,6 +350,8 @@ int run_test(int argc, ACE_TCHAR *argv[])
 
   try
     {
+    DDS::DomainParticipantFactory_var dpf =
```

Comment:
Is this needed?  Not necessarily wrong but this is more of a component-level test that doesn't need the core DDS running.